examples/feeds/create_feed_item_set.py

```diff
@@ -0,0 +1,117 @@
+    """
+    feed_item_set_service = client.get_service(
+        "FeedItemSetService", version="v6"
+    )
+    feed_item_set_operation = client.get_type(
+        "FeedItemSetOperation", version="v6"
+    )
+
+    feed_item_set = feed_item_set_operation.create
+    feed_resource_name = client.get_service(
+        "FeedService", version="v6"
+    ).feed_path(customer_id, feed_id)
+    feed_item_set.feed = feed_resource_name
+    feed_item_set.display_name = "Feed Item Set #%s" % uuid.uuid4()
+
+    """A feed item set can be created as a dynamic set by setting an optional
+    filter field below. If your feed is a location extension, uncomment the code
+    that sets dynamic_location_set_filter. If your feed is an affiliate
+    extension, set dynamic_affiliate_location_set_filter instead.
+    """
+    # 1) Location Extension
+    # NOTE: Does not support Curated location extensions.
+    """
+    dynamic_location_set_filter = feed_item_set.dynamic_location_set_filter
+    business_name_filter = dynamic_location_set_filter.business_name_filter
+    business_name_filter.business_name = 'INSERT_YOUR_BUSINESS_NAME_HERE'
+    business_name_filter.filter_type = client.get_type(
+        "FeedItemSetStringFilterTypeEnum", version="v6"
+    ).EXACT
+    """
+    # 2) Affiliate Extension
+    """
+    dynamic_affiliate_location_set_filter = feed_item_set.dynamic_affiliate_location_set_filter
+    dynamic_affiliate_location_set_filter.chain_ids.extend([INSERT_CHAIN_IDS_HERE])
+    """
+
+    try:
+        response = feed_item_set_service.mutate_feed_item_sets(
+            customer_id, [feed_item_set_operation]
+        )
+        print(
+            "Created a feed item set with resource name: "
+            f"'{response.results[0].resource_name}'"
+        )
+    except GoogleAdsException as ex:
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+if __name__ == "__main__":
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description="Creates a new feed item set for a specified feed."
+    )
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument(
+        "-c",
+        "--customer_id",
+        type=str,
+        required=True,
+        help="The Google Ads customer ID.",
+    )
+    parser.add_argument(
+        "-f", "--feed_id", type=str, required=True, help="The feed ID.",
```

Comment:
use consistent formatting for argument arguments

Reply:
done